x-pack/plugins/synthetics/public/apps/synthetics/components/common/screenshot/empty_image.tsx

```diff
@@ -32,16 +36,21 @@ export const imageStyle = css`
 `;
 
 export const EmptyImage = ({
+  size,
   isLoading = false,
-  width = IMAGE_WIDTH,
-  height = IMAGE_HEIGHT,
+  width,
+  height,
 }: {
   isLoading: boolean;
   width?: number;
   height?: number;
+  size?: EuiImageProps['size'];
 }) => {
   const { euiTheme } = useEuiTheme();
 
+  const imgWidth = width || size === 'm' ? IMAGE_WIDTH_M : IMAGE_WIDTH;
+  const imgHeight = height || size === 'm' ? IMAGE_HEIGHT_M : IMAGE_HEIGHT;
```

Comment:
I assume this is the right intention.

```suggestion
  const imgWidth = width || (size === 'm' ? IMAGE_WIDTH_M : IMAGE_WIDTH);
  const imgHeight = height || (size === 'm' ? IMAGE_HEIGHT_M : IMAGE_HEIGHT);
```

Otherwise when a width or a height is specified [it will always use the `IMAGE_*_M`](https://codesandbox.io/s/vigilant-cartwright-r9cnmi?file=/src/App.js).

Reply:
Maybe it doesn't matter, but what `height` should we apply if a consumer does something like `<EmptyImage size="m" width={100} />` ?

Right now it will give a 100x114 image, which I imagine we don't want. Do we always use the same aspect ratio for the screenshots? If that's the case maybe we can ignore the `size` and calculate the